### PR TITLE
remove uses of go-multicodec

### PR DESCRIPTION
**Pull request by Stebalien**
~Pending deprecation. See: https://github.com/multiformats/go-multicodec/pull/36~

### CODE REVIEW

**Comment by anacrolix** [comment]
https://github.blog/2019-02-14-introducing-draft-pull-requests/

File: echo/main.go [hunk 1]
```diff
@@ -2,7 +2,6 @@ package main
 
 import (
 	"bufio"
-	"context"
```
Comment (anacrolix) [inline echo/main.go]: This is breaking the build and CI.

**Comment by Kubuxu** [comment]
> https://github.blog/2019-02-14-introducing-draft-pull-requests/

@anacrolix this was created long before that feature.

File: echo/main.go (hunk 1 above)
Comment (Stebalien) [reply echo/main.go]: Not sure how that happened. Fixed.

**Comment by anacrolix** [comment]
Note that go mod now fails due to tag rewriting, but it's unrelated to this PR. Travis is still passing because it's cached `go-log`.

**Pull request merged**